mindsdb/integrations/handlers/aqicn_handler/tests/test_aqicn_handler.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+from mindsdb.integrations.handlers.aqicn_handler.aqicn_handler import AQICNHandler
+from mindsdb.integrations.handlers.aqicn_handler.aqicn_tables import *
+from mindsdb.api.executor.data_types.response_type import RESPONSE_TYPE
+
+
+class AQICNHandlerTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.kwargs = {
+            "api_key": "YOUR_API_KEY" # Update this with your aqicn api key
```

Comment:
Since these are unit tests, the API calls have to be mocked. Else all the tests will fail.

Reply:
I think this new commit fixes that.